inboundxml/inboundxml.go

```diff
@@ -24,7 +502,7 @@ type InboundXML struct {
 func (i *InboundXML) GetSchema() ([]byte, error) {
 	b, err := ioutil.ReadFile("./schema/inboundxml.xsd")
```

Comment:
Can you instead put the schema under schema/inboundxml.xsd ? 
Else this line is redundant

Reply:
I faced the issue even when schema file was present. I guess, it works with old go build systems (non-module based builds)